modules/core/src/main/java/org/apache/ignite/internal/LongJVMPauseDetector.java

```diff
@@ -81,17 +93,18 @@ public LongJVMPauseDetector(IgniteLogger log) {
      * Starts worker if not started yet.
      */
     public void start() {
-        if (getBoolean(IGNITE_JVM_PAUSE_DETECTOR_DISABLED, false)) {
+        if (getBoolean(IGNITE_JVM_PAUSE_DETECTOR_DISABLED, DEFAULT_JVM_PAUSE_DETECTOR_DISABLED)) {
             if (log.isDebugEnabled())
                 log.debug("JVM Pause Detector is disabled.");
 
             return;
         }
 
         final Thread worker = new Thread("jvm-pause-detector-worker") {
-            private long prev = System.currentTimeMillis();
 
             @Override public void run() {
+                lastWakeUpTime = System.currentTimeMillis();
```

Comment:
static analyzers will show that lastWakeUpTime used in mixed sync mode, can u prevent this ?